packages/iris-grid/src/IrisGridModel.js

```diff
@@ -186,6 +186,13 @@ class IrisGridModel extends GridModel {
     throw new Error('set customColumns not implemented');
   }
 
+  /**
+   * @param {String[]} columns The columns to treat as frozen
```

Comment:
We should default to `undefined`, and that means just use the layout hints (if they're there), otherwise if we pass an array then those override the layout hints.
```suggestion
   * @param {String[]|undefined columns The columns to treat as frozen, or undefined to just use the layout hints
```